mmd_tools/operators/morph.py

```diff
@@ -227,81 +225,84 @@ class RemoveMaterialOffset(Operator):
     bl_label = 'Remove Material Offset'
     bl_description = ''
     bl_options = {'PRESET'}
-    
+
+    @classmethod
+    def poll(cls, context):
+        root = mmd_model.Model.findRoot(context.active_object)
+        return root and mmd_model.Model(root).firstMesh()
+
```

Comment:
operators can be searched at any time on Blender by pressing space, which could cause the operator to be executed accidentally. Implementing that method should prevent it or at least least Blender handle it and show the proper error message to the user.